src/client/containers/OrderConfirmationPage/OrderConfirmationPage.Container.js

```diff
@@ -2,5 +2,30 @@ import React from 'react';
 import './OrderConfirmationPage.styles.css';
 
 export default function OrderConfirmationPage() {
-  return <div>Order Confirmation Page</div>;
+  return <div className="confirmation-page">
+    <div className="inner-div">
+      <div className="wrapper">
+        <div className="wrapper-content">
+          <h3>
+            THANK YOU FOR YOUR ORDER
+          </h3>
+          <p>
+            We have sent a receipt to your email
+          </p>
+        </div>
+      </div>
+      <div className="wrapper2">
+      <div className="wrapper-content2">
+      <button type='button' onClick={() => {
+            window.location.href = '/';
+          }} className="btn">CONTINUE SHOPPING</button>
+      
+
+<button type='button' onClick={() => {
+            window.location.href = '/';
```

Comment:
Should this action also bring you back to the start page, I would expect it to go somewhere else.

Reply:
start page is '/'. That's why.. Thanks

Reply:
Yes, but with the code as it is now, both buttons bring you back to the same page?
Then I don't understand why there are two buttons :)